### PR TITLE
Rating stars refactor + sketch generated stars

### PR DESCRIPTION
**Pull request by oddcelot**

### CODE REVIEW

File: packages/components-sketch/app.js [hunk 1]
```diff
@@ -32,8 +33,10 @@ app.get('/', (req, res) => {
 
 app.get('/:component', (req, res) => {
   try {
-    res.render(req.params.component, {
-      title: req.params.component.replace(/^./, c => c.toUpperCase()),
+    res.render(sanitize(req.params.component), {
+      title: sanitize(req.params.component)
+        .replace(/-/g, ' ')
+        .replace(/\b\w/g, (c) => c.toUpperCase()),
```
Comment (acstll) [inline packages/components-sketch/app.js:39]: Feels good 😄

**Review by acstll — COMMENTED** [review]
Looks good to merge. We need to wait for a11y approval before we merge this, so we could close this and re-open when the time comes.

Would it be possible to separate the "sketch" part in a different PR, or is it too closely related?

File: packages/components-sketch/app.js (hunk 1 above)
Comment (oddcelot) [reply packages/components-sketch/app.js:39]: > Would it be possible to separate the "sketch" part in a different PR, or is it too closely related?
this is intertwined, so not really 😅

**Comment by acstll** [comment]
Exceptionally merging before getting the a11y feedback to save us some trouble in the future 😑

**Pull request merged**